autoprofile.py

```diff
@@ -33,32 +36,107 @@ def __init__(self):
         self.name = _("Automatic Profile")
         self.bio_enabled = False
         self.name_enabled = False
+        self.pfp_enabled = False
+        self.pfp_remove_latest = None
         self.raw_bio = None
         self.raw_name = None
+        self.pfp_degree = 0
 
     async def client_ready(self, client, db):
         self.client = client
 
+    async def autopfpcmd(self, message):
+        """Rotates your profile picture every n seconds with x degrees, usage:
+           .autopfp <timeout> <degrees> <remove previous (last pfp)>
+
+           Timeout - seconds
+           Degrees - 60, -10, etc
+           Remove last pfp - True/False, case sensitive"""
+
+        if not await self.client.get_profile_photos(await self.client.get_me(), limit=1):
+            return await utils.answer(message, _("<b>You don't have profile pic set.</b>"))
```

Comment:
...have **a** profile pic...

Reply:
Why not just .title() the arguement, makes it NOT case sensitive :P